test/e2e-test-application/e2e/tests/1-angular/luigi-client-link-manager-features.spec.js

```diff
@@ -562,4 +562,103 @@ describe('Luigi client linkManager', () => {
+      });
+    });
+
+    it('open webcomponent timer', () => {
+      cy.window().then(win => {
+        cy.wait(500);
+        cy.get('.wcContainer>div>div>*').then(container => {
+          const root = container.children().prevObject[1].shadowRoot;
+          const wcContent = root.querySelector('p').innerText;
+
+          expect(wcContent).to.equal('0');
+        });
+      });
+    });
+
+    it('click on webcomponent btn', () => {
+      cy.window().then(win => {
+        cy.wait(500);
+        cy.get('.wcContainer>div>div>*').then(container => {
+          const root = container.children().prevObject[0].shadowRoot;
+          root.querySelector('button').click();
+          const wcContent = root.querySelector('button').innerText;
+
+          expect(wcContent).to.equal('Stop');
+        });
+      });
+    });
+
+    it('listener on webcomponent timer', () => {
+      cy.window().then(win => {
+        cy.wait(500);
+        cy.get('.wcContainer>div>div>*').then(container => {
+          const rootBtn = container.children().prevObject[0].shadowRoot;
+          rootBtn.querySelector('button').click();
+          const root = container.children().prevObject[1].shadowRoot;
+          const wcContent = root.querySelector('p').innerText;
+
+          expect(wcContent).to.equal('1');
+        });
+      });
+    });
+
+    it('click start timer on  webcomponent btn and reaction in webcomponent timer', () => {
+      cy.window().then(win => {
+        cy.wait(500);
+        cy.get('.wcContainer>div>div>*').then(container => {
+          const rootBtn = container.children().prevObject[0].shadowRoot;
+          const wcContentStart = container.children().prevObject[1].shadowRoot.querySelector('p').innerText;
+          rootBtn.querySelector('button').click();
+          const wcContent = rootBtn.querySelector('button').innerText;
+          expect(wcContent).to.equal('Stop');
+
+          cy.wait(500);
+          const wcContentStop = container.children().prevObject[1].shadowRoot.querySelector('p').innerText;
+          expect(wcContentStart).to.not.equal(wcContentStop);
+        });
+      });
+    });
+
+    it('click start and stop timer on webcomponent btn and reaction in webcomponent timer', () => {
+      cy.window().then(win => {
+        cy.wait(500);
+        cy.get('.wcContainer>div>div>*').then(container => {
+          const rootBtn = container.children().prevObject[0].shadowRoot;
+          rootBtn.querySelector('button').click();
+          cy.wait(500);
+
+          rootBtn.querySelector('button').click();
+          const wcContent = rootBtn.querySelector('button').innerText;
+          expect(wcContent).to.equal('Start');
+
+          const wcContentStart = container.children().prevObject[1].shadowRoot.querySelector('p').innerText;
+          cy.wait(500);
+          const wcContentStop = container.children().prevObject[1].shadowRoot.querySelector('p').innerText;
+          expect(wcContentStart).to.equal(wcContentStop);
```

Comment:
```suggestion

          const wcContentStart = container.children().prevObject[1].shadowRoot.querySelector('p').innerText;
  
          rootBtn.querySelector('button').click();
          const wcContent = rootBtn.querySelector('button').innerText;
          expect(wcContent).to.equal('Start');

          cy.wait(500);
          const wcContentStop = container.children().prevObject[1].shadowRoot.querySelector('p').innerText;
          expect(wcContentStart).to.not.equal(wcContentStop);
```

The **wcContentStart** and **wcContentStop** are identical below: There is no change in between so the **expect equal** does not check 2 different states but same variable.
```
const wcContentStart = container.children().prevObject[1].shadowRoot.querySelector('p').innerText;
cy.wait(500);
const wcContentStop = container.children().prevObject[1].shadowRoot.querySelector('p').innerText;
expect(wcContentStart).to.equal(wcContentStop); // checking same variable <---
```

 You might also need to at least put **wcContentStart** somewhere up above. and check for 
`expect(wcContentStart).to.not.equal(wcContentStop);`